util/cron/test-hpe-apollo-hdr.arkouda.bash

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Run arkouda correctness testing on a hpe-apollo with HDR IB
+
+UTIL_CRON_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
+export ARKOUDA_URL=https://github.com/e-kayrakli/arkouda.git
+export ARKOUDA_BRANCH=server-util-pbs
```

Comment:
Note that these will be removed once that branch is merged.

Reply:
Just noting that there are a few other places which need to be changed.